### PR TITLE
Add veBAL table to portfolio page

### PR DESCRIPTION
**Pull request by sssmi**
# Description

Addresses an issue: https://github.com/balancer-labs/frontend-v2/issues/1695

- Remove "Incl." text from veBal banner
- Add veBAL protocol liquidity table with Expiry date column to portfolio page
- Rename "Stake" column of PoolsTable to "Actions" column. Add lock/stake action button.

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## How should this be tested?

Please provide instructions so we can test. Please also list any relevant details for your test configuration.

- Your locked veBal show on the "veBAL protocol liquidity" table
- Not-locked veBal should show on the "Unstaked pools" table and have a "Lock" action

## Visual context

| Portfolio Page Now |  Portfolio Page  Before |
| -------------- | -------------- |
|   Screen Shot 2022-06-29 at 12 17 26 |  Screen Shot 2022-06-29 at 15 38 48 |

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have requested at least 2 reviews (If the PR is significant enough, use best judgement here)
- [x] I have commented my code where relevant, particularly in hard-to-understand areas
- [x] The base of this PR is `master` if hotfix, `develop` if not

### CODE REVIEW

File: src/components/contextual/pages/pools/VeBalPoolTable.vue [hunk 1]
```diff
@@ -0,0 +1,63 @@
+<script setup lang="ts">
+import { computed } from 'vue';
+
+import PoolsTable from '@/components/tables/PoolsTable/PoolsTable.vue';
+import { useLock } from '@/composables/useLock';
+import { VeBalLockInfo } from '@/services/balancer/contracts/contracts/veBAL';
+import { Pool, PoolWithShares } from '@/services/pool/types';
+
+/**
+ * PROPS & EMITS
```
Comment (alter-eggo) [inline src/components/contextual/pages/pools/VeBalPoolTable.vue:10]: seems there are no emits here

File: src/components/tables/PoolsTable/PoolsTableActionsCell.vue [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+ */
+const props = withDefaults(defineProps<Props>(), {});
+
+const emit = defineEmits<{
+  (e: 'click:stake', value: PoolWithShares): void;
+}>();
+
+/**
+ * COMPOSABLES
+ */
```
Comment (alter-eggo) [inline src/components/tables/PoolsTable/PoolsTableActionsCell.vue:26]: redundant comment

File: src/components/contextual/pages/pools/UnstakedPoolsTable.vue [hunk 3]
```diff
@@ -104,7 +104,7 @@ const poolsToRender = computed(() => {
 });
 
 const hiddenColumns = computed((): string[] => {
-  const hiddenCols = ['poolVolume', 'poolValue', 'migrate'];
+  const hiddenCols = ['poolVolume', 'poolValue', 'migrate', 'lockedEndDate'];
```
Comment (alter-eggo) [inline src/components/contextual/pages/pools/UnstakedPoolsTable.vue:107]: there is no need in computed function here

File: src/components/contextual/pages/pools/VeBalPoolTable.vue [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+          props.lock?.hasExistingLock && !props.lock?.isExpired
+            ? props.lock?.lockedEndDate
+            : undefined
+      }
+    ];
+  }
+  return [];
+});
+
+const hiddenColumns = computed(() => {
```
Comment (alter-eggo) [inline src/components/contextual/pages/pools/VeBalPoolTable.vue:39]: same, no need in computed here

**Review by alter-eggo — CHANGES_REQUESTED** [review]

**Comment by timjrobinson** [comment]
I'm seeing "Stake" instead of lock on the 80/20 pool on Goerli. 

2022-07-01-155551_1584x64_scrot

File: src/components/tables/PoolsTable/PoolsTable.vue [hunk 5]
```diff
@@ -166,6 +168,14 @@ const columns = computed<ColumnDefinition<PoolWithShares>[]>(() => [
     },
     width: 250
   },
+  {
+    name: t('expiryDate'),
+    Cell: 'lockedEndDateCell',
+    accessor: 'lockedEndDate',
+    align: 'right',
+    id: 'lockedEndDate',
```
Comment (timjrobinson) [inline src/components/tables/PoolsTable/PoolsTable.vue:176]: `lockEndDate` sounds more correct in english.

File: src/components/tables/PoolsTable/PoolsTable.vue [hunk 6]
```diff
@@ -215,6 +223,10 @@ function aprLabelFor(pool: PoolWithShares): string {
 
   return totalAprLabel(poolAPRs, pool.boost);
 }
+
+function lockedUntil(lockedEndDate?: number) {
+  return lockedEndDate ? format(lockedEndDate, PRETTY_DATE_FORMAT) : '—';
```
Comment (timjrobinson) [inline src/components/tables/PoolsTable/PoolsTable.vue:228]: Same here with `lockEndDate`

**Comment by pkattera** [comment]
> I'm seeing "Stake" instead of lock on the 80/20 pool on Goerli.
> 
> 2022-07-01-155551_1584x64_scrot

@timjrobinson I also see 'Stake' on Goerli and Kovan. I believe this is due a bug due to that specific pool being set up incorrectly on Kovan. 

I have tested this on Mainnet and I see the correct 'Lock' button:

Screen Shot 2022-07-01 at 10 40 01 AM

**Review by timjrobinson — APPROVED** [review]

**Pull request merged**